app/routes/users/components/UserSettings.tsx

```diff
@@ -91,112 +96,116 @@ const UserSettings = () => {
 
   const showAbakusMembership = user.isStudent;
 
-  const onSubmit = (values: FormValues) =>
-    dispatch(updateUser(values)).then(() => {
+  const onSubmit = (values: FormValues) => {
+    const body = {
+      ...values,
+      gender: values.gender.value,
+    };
+
+    dispatch(updateUser(body)).then(() => {
       navigate('/users/me');
     });
+  };
 
-  const initialValues = {
+  const initialValues: FormValues = {
     ...user,
+    gender: { label: Gender[user.gender], value: user.gender },
     isAbakusMember: user?.isAbakusMember.toString(),
   };
 
   return (
     <>
-      <div className={styles.pictureSection}>
-        <UserImage user={user} />
-      </div>
-
-      <RemovePicture username={user.username} />
-
       <TypedLegoForm
         onSubmit={onSubmit}
         initialValues={initialValues}
         validate={validate}
         subscription={{}}
       >
         {({ handleSubmit }) => (
-          <Form onSubmit={handleSubmit}>
-            <Field
-              placeholder="Brukernavn"
-              label="Brukernavn"
-              name="username"
-              readOnly
-              component={TextInput.Field}
-              disabled
-            />
+          <form onSubmit={handleSubmit}>
```

Comment:
Quick question 😁 Why did this get turned into the default form?

Reply:
Because the `Form` includes some flex styling, but it ended up not really mattering